### PR TITLE
Fix a problem in WBDevice.swift

### LINKED ISSUE #44: Problem in WBDevice.swift
Value of optional type 'CBService?' must be unwrapped to refer to member 'uuid' of wrapped base type 'CBService'

Fix: Chain the optional using '?' to access member 'uuid' only for non-'nil' base values
Fix: Force-unwrap using '!' to abort execution if the optional value contains 'nil'

### PR DESCRIPTION
**Pull request by googol7**
Fix #44

Fix made by https://github.com/niklasfessler

### CODE REVIEW

File: WBCore/WBDevice.swift [hunk 1]
```diff
@@ -106,7 +106,11 @@ open class WBDevice: NSObject, Jsonifiable, CBPeripheralDelegate {
             super.init(transaction: transaction)
         }
         func matchesCharacteristic(_ characteristic: CBCharacteristic) -> Bool {
-            return self.serviceUUID == characteristic.service.uuid && self.characteristicUUID == characteristic.uuid
+            guard
+                let uuid = characteristic.service?.uuid else {
```
Comment (daphtdazz) [inline WBCore/WBDevice.swift:110]: could you rename this `serviceUUID` so that it's a bit clearer lower down?

Comment (daphtdazz) [reply WBCore/WBDevice.swift]: well that was quick!

**Comment by daphtdazz** [comment]
@googol7 is this a problem for you using WebBLE, would you like me to make a bugfix release for it?

**Comment by googol7** [comment]
@daphtdazz we forked this repo. So it’s not a problem for us.

**Pull request merged**